counter/coco.nut

```diff
@@ -1,20 +1,22 @@
 [coco]
-version = "0.3.0"
+version = "0.2.0"
 
 [module]
 name = "Counter"
-version = "1.0.0"
-license = "MIT"
-repository = "https://github.com/sarvalabs/cocolang-examples/counter"
-authors = ""
+version = "0.0.1"
```

Comment:
please update the `coco.nut` file with values from the old one. such as the version, licence and repository links. also add your name and mine as authors

Reply:
Made necessary changes